sql/mysql/sqlspec.go

```diff
@@ -322,13 +322,15 @@ func integerSpec(t *schema.IntegerType) (*sqlspec.Column, error) {
 		return &sqlspec.Column{TypeName: "int"}, nil
 	case tTinyInt:
 		return &sqlspec.Column{TypeName: "int8"}, nil
+	case tMediumInt:
+		return &sqlspec.Column{TypeName: "int24"}, nil
```

Comment:
What is `int24`?

Reply:
https://dev.mysql.com/doc/c-api/8.0/en/c-api-prepared-statement-type-codes.html

it's a type in mysql